homework/02LessonTwo.js

```diff
@@ -53,22 +69,22 @@ function lessThanOrEqualTo(parameter1, parameter2) {
 //
 // What is a function? How do you define a function in Javascript?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: A function is a relationship between two parameters. From w3schools: "A JavaScript function is defined with the function keyword, followed by a name, followed by parentheses ()."
 //
```

Comment:
You have the right mindset here...simply put, a function is a block of code designed to perform a task.